deploy/kubernetes/webhook-example/admission-configuration-template

```diff
@@ -8,7 +8,7 @@ webhooks:
   - apiGroups:   ["snapshot.storage.k8s.io"]
     apiVersions: ["v1", "v1beta1"]
     operations:  ["CREATE", "UPDATE"]
-    resources:   ["volumesnapshots", "volumesnapshotcontents"]
+    resources:   ["volumesnapshots", "volumesnapshotcontents", "volumesnapshotclasses"]
```

Comment:
Please add this to the release note. Any time we add a new RBAC rule, it requires a major version bump.

Reply:
This is not RBAC rule. But I added some text about it to the first commit.